src/omotes_orchestrator/sdk_interface.py

```diff
@@ -165,7 +172,9 @@ def send_job_progress_update(self, job: Job, progress_update: JobProgressUpdate)
         :param progress_update: Current progress for the job.
         """
         self.broker_if.send_message_to(
-            OmotesQueueNames.job_progress_queue_name(job), progress_update.SerializeToString()
+            exchange_name=OmotesQueueNames.omotes_exchange_name(),
+            routing_key=OmotesQueueNames.job_progress_queue_name(job),
```

Comment:
Maybe indicating there are these `routing_key` (or even exchange, but that might become a bit too busy in the diagram) in the Queue design document can help? Just like `routing_key=OmotesQueueNames.available_workflows_routing_key()` is indicated in the Queue design diagram.
 
https://github.com/Project-OMOTES/architecture-documentation?tab=readme-ov-file#queue-design

Just as an optional addition, but overall the diagram really helps!

Reply:
Ahh great catch, I forgot to update the queue design diagram with the new exchanges and routing keys. Good idea!
I have put in everything just to see how busy it would get. What do you think? https://github.com/Project-OMOTES/architecture-documentation/blob/main/Technical%20Architecture/Queue%20design.v1.drawio.png

Reply:
Looks good! Thanks!